app/views/tracks/exercises/show.html.haml

```diff
@@ -3,85 +3,39 @@
+              = render "tracks/exercises/show/timeline_iterated", track: @track, exercise: @exercise, solution: @solution, user_track: @user_track
+            - elsif @solution.status_started?
+              = render "tracks/exercises/show/timeline_pending", track: @track, exercise: @exercise, solution: @solution
+
+          - else
+            - if @user_track.exercise_unlocked?(@exercise)
+              = render "tracks/exercises/show/timeline_pending", track: @track, exercise: @exercise, solution: nil
+            - elsif @user_track.tutorial_exercise_completed?
+              = render "tracks/exercises/show/action_box_locked", exercise: @exercise, user_track: @user_track
             - else
-              Explore the #{external_link_to 'source of this exercise', source_url}.
-      %section.extra-info
-        = render ReactComponents::Track::ExerciseMakersButton.new(@exercise)
-
-        = link_to "https://github.com/exercism/#{@track.slug}/tree/main/exercises/#{@exercise.git_type}/#{@exercise.slug}", target: "_blank", rel: 'noreferrer' do
-          = graphical_icon "external-site-github", css_class: "github-icon"
-          %span Edit via GitHub
-          = icon "external-link", "The link opens in a new window or tab", css_class: "action-icon"
-
-    .flex-shrink-0.lg:w-1-3.lg:ml-32.mb-32.lg:mb-0
-      - if @solution
-        - if @solution.status_completed? || @solution.status_published?
-          = render "tracks/exercises/show/action_box_completed", track: @track, exercise: @exercise, solution: @solution
-        - elsif @solution.status_iterated?
-          = render "tracks/exercises/show/action_box_iterated", track: @track, exercise: @exercise, solution: @solution, user_track: @user_track
-        - elsif @solution.status_started?
-          = render "tracks/exercises/show/action_box_started", track: @track, exercise: @exercise, solution: @solution
+              = render "tracks/exercises/show/action_box_locked_by_hello_world", exercise: @exercise, user_track: @user_track, track: @track
 
-      - elsif !@user_track.external?
-        - if @user_track.exercise_unlocked?(@exercise)
-          = render "tracks/exercises/show/action_box_start", track: @track, exercise: @exercise
-        - elsif @user_track.tutorial_exercise_completed?
-          = render "tracks/exercises/show/action_box_locked", exercise: @exercise, user_track: @user_track
-        - else
-          = render "tracks/exercises/show/action_box_locked_by_hello_world", exercise: @exercise, user_track: @user_track, track: @track
+        .mt-28.ml-8
+          = render ViewComponents::ProminentLink.new("View track documentation", Exercism::Routes.track_docs_path(@track))
 
-      - elsif user_signed_in?
-        = render "tracks/exercises/show/action_box_join_track", exercise: @exercise, user_track: @user_track
+        /= ReactComponents::Student::Nudge.new(@solution) if @solution
+        /= ReactComponents::Student::SolutionSummary.new(@solution) if @solution
```

Comment:
Maybe delete SolutionSummary altogether since it is no longer used?

Reply:
Yep - do it!